src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -263,7 +263,8 @@ internal static string FindBestVersionMatch(VersionRange versionRange, IEnumerab
                 return version;
             }).Where(v => v != null);
 
-            var matchingVersion = bundleVersions.OrderByDescending(version => version.Version).FirstOrDefault();
+            var latestBundleVersion = bundleVersions.OrderByDescending(version => version.Version).FirstOrDefault();
```

Comment:
Can we move defaulting to this into `ResolvePlatformReleaseChannelVersion`? so we only fallback to it if it is needed.